src/app/ActivityConfig.ts

```diff
@@ -1,47 +1,60 @@
-class Activity {
+export class Activity {
   name: string;
   url: string;
+  levels: number;
 }
 
 export const ActivityConfig: Array<Activity> = [
   {
     name: 'Highlight The Definition',
-    url: '/introduction/activity-1'
+    url: '/introduction/activity-1',
+    levels: 1
```

Comment:
Have another file where you define the number of levels as constants and then refer it here..